### DISCUSSION TITLE
v3.2.88

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#13187] [General]
## What's Changed
* Escape `$recordGroupTitle` when passed to Alpine in table view by @rossbearman in https://github.com/filamentphp/filament/pull/13178
* Fix for bug introduced in #13074: Shift+Click to select records breaks spa mode by @neverything in https://github.com/filamentphp/filament/pull/13186

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.87...v3.2.88

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.88'>v3.2.88</a>.</em>

**Comment by fatihdurmaz** [comment]
Rich Editor not resetting error continues.
